lib/generamba/service/remote_plugin.rb

```diff
@@ -0,0 +1,51 @@
+module Generamba
+  module Service
+    class RemotePlugin
```

Comment:
I really need comments for classes and methods, otherwise I can't understand the purpose of some classes :)